code/modules/mob/living/simple_animal/abnormality/zayin/bald.dm

```diff
@@ -25,37 +25,40 @@
 	gift_chance = 10
 	gift_message = "Now we're feeling awesome!"
 
-	var/list/balding_list = list(
-							"Bald"
-							)
+	var/list/balding_list = list("Bald")
 	var/bald_users = list()
 
 /mob/living/simple_animal/hostile/abnormality/bald/WorkChance(mob/living/carbon/human/user, chance)
-	if(user.hairstyle in balding_list)
+	if(HAS_TRAIT(user, TRAIT_BALD))
 		return 95
 	return chance
 
+/mob/living/simple_animal/hostile/abnormality/bald/WorktickSuccess(mob/living/carbon/human/user)
+	if(HAS_TRAIT(user, TRAIT_BALD))
+		user.adjustSanityLoss(user.maxSanity * 0.05) // Half of sanity restored for bald people
```

Comment:
This is 5% not 50% if I'm not mistaken.

Reply:
> This is 5% not 50% if I'm not mistaken.

It's 5% per success tick, which we'll be having 10 of.